releasenotes/notes/docker-multiline-support-84adf327b76bf6e0.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Added an abstraction to unwrap logs in the MultilineHandler to support multi-line with Docker.
```

Comment:
What are those release notes for? is it for the changelog? Should it be more user oriented?
I'd go for `fixed issue with multi-line tailing of docker containers output`, as users don't care much about an added abstraction

Reply:
It's for user, agreed.